### PR TITLE
chore: fix storybook menu alignment and order

### PR DESCRIPTION
**Pull request by denieler**
### Description

Fix Picasso storybook menu alignment and order of the sections

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Screenshot 2021-04-28 at 12 55 10 | Screenshot 2021-04-28 at 13 00 23 |
| Screenshot 2021-04-28 at 12 55 38 | Screenshot 2021-04-28 at 12 59 58 |

### Review

- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/api-principles.md)
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
- [x] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

### CODE REVIEW

File: .storybook/load-stories.js [hunk 1]
```diff
@@ -27,9 +29,10 @@ PicassoBook.addSections([
   'Widgets'
 ])
 
-require('./stories/Picasso') // markdown pages for README & CHANGELOG
-require('./stories/Contributing') // markdown pages for contribution guide
-reqStorybook.keys().forEach(filename => reqStorybook(filename))
+reqStorybook.keys().forEach(filename => {
+  console.log(filename)
```
Comment (teimurjan) [inline .storybook/load-stories.js:33]: Shouldn't it be deleted?

**Review by teimurjan — APPROVED** [review]
👍

File: .storybook/load-stories.js (hunk 1 above)
Comment (denieler) [reply .storybook/load-stories.js]: yep, thank you 🙇 removed

**Comment by denieler** [comment]
@teimurjan @havenchyk guys, there is also one difference with the order of Readme and Changelog sections:

Screenshot 2021-04-28 at 13 04 55

vs

Screenshot 2021-04-28 at 13 04 48

not sure, how important it is. But if you guys think it is - I can also more them up.

**Comment by havenchyk** [comment]
@denieler I would keep readme in the top if possible

**Comment by denieler** [comment]
@havenchyk done 👍 
Screenshot 2021-04-28 at 13 24 45

File: .storybook/components/PicassoBook/Book.js [hunk 2]
```diff
@@ -51,19 +58,29 @@ in '.storybook/config.js'.
...
     // {
     //   'Tutorials': 0,
-    //   'Components': 1,
+    //   'Components': 2,
     //   ...
     // }
     const sectionsOrder = {}
     for (let index in this.sections) {
-      sectionsOrder[this.sections[index]] = Number(index)
+      sectionsOrder[this.sections[index]] = Number(index) * 2
```
Comment (havenchyk) [inline .storybook/components/PicassoBook/Book.js:66]: why *2?

File: .storybook/manager-head.html [hunk 3]
```diff
@@ -1,3 +1,9 @@
+<style>
+  .sidebar-subheading-action {
```
Comment (havenchyk) [inline .storybook/manager-head.html:2]: what Is this for?

**Review by havenchyk — APPROVED** [review]

File: .storybook/components/PicassoBook/Book.js (hunk 2 above)
Comment (denieler) [reply .storybook/components/PicassoBook/Book.js:66]: to have section indexes: `0, 2, 4, 6`
so in each section I would have additional index to put `alwaysOnTop` items. For example in section with number 4 the index of `alwaysOnTop` item would be 3 😃

File: .storybook/manager-head.html (hunk 3 above)
Comment (denieler) [reply .storybook/manager-head.html:2]: Storybook has a section heading icon, which doesn't work in our case, so I've hided it

**Pull request merged**